library/core/src/mem/manually_drop.rs

```diff
@@ -56,9 +56,9 @@ use crate::ptr;
 /// working with [pinned] data, where reusing the memory without calling the destructor could lead
 /// to Undefined Behaviour.
 ///
-/// [`mem::zeroed`]: fn.zeroed.html
-/// [`MaybeUninit<T>`]: union.MaybeUninit.html
-/// [pinned]: ../pin/index.html
+/// [`mem::zeroed`]: crate::mem::MaybeUninit::zeroed
```

Comment:
Are you sure this didn't mean to link to `crate::mem::zeroed`?
```suggestion
/// [`mem::zeroed`]: crate::mem::zeroed
```